content/en/docs/setup/production-environment/tools/kubeadm/high-availability.md

```diff
@@ -100,31 +100,17 @@ option. Your cluster requirements may need a different configuration.
 
 ### Steps for the first control plane node
 
-1.  On the first control plane node, create a configuration file called `kubeadm-config.yaml`:
-
-        apiVersion: kubeadm.k8s.io/v1beta2
-        kind: ClusterConfiguration
-        kubernetesVersion: stable
-        controlPlaneEndpoint: "LOAD_BALANCER_DNS:LOAD_BALANCER_PORT"
-
-    - `kubernetesVersion` should be set to the Kubernetes version to use. This
-  example uses `stable`.
-    - `controlPlaneEndpoint` should match the address or DNS and port of the load balancer.
-    - It's recommended that the versions of kubeadm, kubelet, kubectl and Kubernetes match.
-
-{{< note >}}
-Some CNI network plugins like Calico require a CIDR such as `192.168.0.0/16` and
-some like Weave do not. See the [CNI network documentation](/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm/#pod-network).
-To add a pod CIDR set the `podSubnet: 192.168.0.0/16` field under
-the `networking` object of `ClusterConfiguration`.
-{{< /note >}}
-
 1.  Initialize the control plane:
 
     ```sh
-    sudo kubeadm init --config=kubeadm-config.yaml --upload-certs
+    sudo kubeadm init --kubernetes-version stable --control-plane-endpoint "LOAD_BALANCER_DNS:LOAD_BALANCER_PORT" --upload-certs
```

Comment:
If I'm not wrong, we can remove `--kubernetes-version` stable (it is the default)